### ISSUE TITLE
Single channel DCCRN speech enhancement model

### ISSUE THREAD

**Issue by ndisci** [espnet/espnet#4930]
Hello,
I have a question about SE models. I want to get DCCRN single channel model that has the best pesq value for CHiMe4 dataset but couldn't see it in Hugging Face models page.
Thanks.

**Comment by Emrys365** [comment]
Hi @ndisci, we don't have a pretrained DCCRN model on CHiME4, but there is one pretrained on DNS20 data:

https://huggingface.co/Johnson-Lsx/Shaoxiong_Lin_dns_ins20_enh_enh_train_enh_dccrn_raw

**Comment by ndisci** [comment]
I've already tried that. Thank you ! :)

**Comment by HyacinthJingjing** [comment]
Hello, I have a question about the pretrained DCCRN model, 
https://huggingface.co/Johnson-Lsx/Shaoxiong_Lin_dns_ins20_enh_enh_train_enh_dccrn_raw
When I use the pretrained model to enhance my test wav, i noticed the magnitude of the enhanced wav is enlarged. Is that normal?
I have run the script in egs2/dns_ins20/enh1/run.sh.
sample_rate=16k
train_set=tr_synthetic
valid_set=test9219
test_sets="testtoy"
#test_sets="tt_synthetic_no_reverb tt_synthetic_with_reverb"

./enh.sh \
    --lang en \
    --train_set "${train_set}" \
    --valid_set "${valid_set}" \
    --test_sets "${test_sets}" \
    --fs ${sample_rate} \
    --ngpu 2 \
    --skip_data_prep true \
    --skip_train true \
    --ref_num 1 \
    --local_data_opts "" \
    --enh_config ./conf/tuning/train_enh_dccrn.yaml \
    --use_dereverb_ref false \
    --use_noise_ref true \
    --max_wav_duration 31 \
    --inference_model "valid.loss.best.pth" \
    "$@"

**Comment by HyacinthJingjing** [comment]
@Emrys365

**Comment by HyacinthJingjing** [comment]
Thank you for you reply. I would like to try this method. Can i train the model with multiple criterions, SI-SNR and MSE loss? Can this method mitigate the issue?

**Comment by Emrys365** [comment]
> Thank you for you reply. I would like to try this method. Can i train the model with multiple criterions, SI-SNR and MSE loss? Can this method mitigate the issue? @Emrys365 

Yes I will expect that multi-task learning should mitigate this issue.

**Comment by HyacinthJingjing** [comment]
Hi @Emrys365 , I have tried to retrain the dccrn model with the SNR loss, but the results didnot meet expectations.
Here is the config file. 
optim: adam
init: null   # do not set init method here because DCCRN has its own initialization
max_epoch: 100
batch_type: folded
batch_size: 32
iterator_type: chunk
chunk_length: 64000
num_workers: 4
optim_conf:
    lr: 1.0e-03
    eps: 1.0e-08
    weight_decay: 1.0e-7
patience: 10
val_scheduler_criterion:
- valid
- loss
best_model_criterion:
-   - valid
    - si_snr
    - max
-   - valid
    - loss
    - min
keep_nbest_models: 20
scheduler: reducelronplateau
scheduler_conf:
    mode: min
    factor: 0.7
    patience: 1
model_conf:
    loss_type: si_snr
encoder: stft
encoder_conf:
    n_fft: 512
    win_length: 400
    hop_length: 100
decoder: stft
decoder_conf:
    n_fft: 512
    win_length: 400
    hop_length: 100
separator: dccrn

criterions:
  # The first criterion
  - name: snr
    conf:
      eps: 1.0e-7
    # the wrapper for the current criterion
    # for single-talker case, we simplely use fixed_order wrapper
    wrapper: fixed_order
    wrapper_conf:
      weight: 1.0
![降噪效果评估](https://github.com/espnet/espnet/assets/17040205/83deaf9e-c469-4458-8dca-da22b5232e35)
My expectation was that the enhanced speech would behave in the same amplitude as the original clean speech. Do you have some idea to solve this issue?

**Comment by Emrys365** [comment]
By default the enhanced audios in stage 7 in [enh.sh](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/enh.sh#L81) will be scale-normalized.

Could you try removing the argument in https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/enh.sh#L81?

**Comment by HyacinthJingjing** [comment]
Yes, I have already tried that. Thank you. 
What situation is this scale-normalization designed for?

**Comment by Emrys365** [comment]
> Yes, I have already tried that. Thank you. What situation is this scale-normalization designed for? @Emrys365

It is to avoid clipping when saving the enhanced audio (values larger than 1.0 or smaller than -1.0 will be clipped if we do not normalize the scale).
